requirements/base.txt

```diff
@@ -16,7 +16,7 @@ urllib3==1.26.15
 # Inherited a4-core requirements
 Django==3.2.19
 django-allauth==0.54.0
-git+https://github.com/liqd/django-autoslug.git@liqd2212#egg=django-autoslug
```

Comment:
Looks like a4-core also uses this dep., I'll initiate a PR there as well so that we can archive/delete (if needed) [liqd/django-autoslug](https://github.com/liqd/django-autoslug) once bother the PRs are merged.

Reply:
Done! (Initiated a PR https://github.com/liqd/adhocracy4/pull/1413)